pyparsing/diagram/__init__.py

```diff
@@ -355,6 +368,15 @@ def _to_diagram_element(
     elif isinstance(element, pyparsing.Empty) and not element.customName:
         # Skip unnamed "Empty" elements
         ret = None
+    elif _is_named_terminal(element):
+        if not _already_extracted(element, lookup):
+            ret = EditablePartial.from_call(
+                railroad.Terminal, title=element.name, text=element.defaultName
+            )
+        else:
+            ret = EditablePartial.from_call(
+                railroad.NonTerminal, title=element.name, text=element.name
```

Comment:
`title` will only set the hover text, I assume you know that? This will make it harder to see both the `customName` and the `defaultName` compared to my current solution of using a group to do that: https://github.com/pyparsing/pyparsing/blob/1b86c4915003ed6bd9d47126f9337801be3853b0/pyparsing/diagram/__init__.py#L387-L393

Reply:
The default name gets displayed in the element's subdiagram, so we shouldn't need to show both defaultName and customName at the same time.

Reply:
Oh, okay. I think I need to see an example of this.